test/blackbox-tests/test-cases/pkg/slang.t

```diff
@@ -8,7 +8,10 @@
 Helper function to create a lockfile with a given install action and then build it, running the action.
   $ test_action() {
   >   dune clean || true
-  >   echo "(install $1)" > dune.lock/test.pkg
+  >   cat > dune.lock/test.pkg <<EOF
+  > (version 0.0.1)
+  > (install $1)
```

Comment:
Could be swapped to avoid the renumbering in the output, but this is just nit-picking.

Reply:
I did that at first but changed it to have the version first because I wanted the lockfiles in the tests to be more realistic.